odoo/http.py

```diff
@@ -402,6 +402,19 @@ def dispatch_rpc(service_name, method, params):
         return dispatch(method, params)
 
 
+def get_session_max_inactivity(env):
+    if not env or env.cr._closed:
+        return SESSION_LIFETIME
+
+    ICP = env['ir.config_parameter'].sudo()
+
+    try:
+        return min(int(ICP.get_param('sessions.max_inactivity_seconds', SESSION_LIFETIME)), SESSION_LIFETIME)
```

Comment:
Now that I see this I'm thinking that maybe we shouldn't allow negative values either:
```suggestion
        value = ICP.get_param('sessions.max_inactivity_seconds', SESSION_LIFETIME)
        return max(0, min(int(value), SESSION_LIFETIME))
```